EIPS/eip-5320.md

```diff
@@ -0,0 +1,158 @@
+---
+eip: 5320
+title: Harberger Tax NFT
+author: Green (@greenlucid)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-23
+discussions-to: todo
+---
+
+### Simple Summary
+
+A standard interface for Non-Fungible Tokens subject to Harberger Taxes.
+
```

Comment:
Not a valid section.  Please see https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for the current layout.